base_multi_image/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2009 Sharoon Thomas Open Labs Business Solutions
+# © 2015 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
I'm not included, for example, and as said, Sharoon Thomas has nothing to do here because he was the original author of a module for multiple images, not this one, that has been totally redesigned, and I added the possibility of loading from URL, db, file...